doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -286,6 +286,11 @@ Libraries
 
 This section provides detailed lists of changes by :ref:`library <libraries>`.
 
+* Added:
+
+  * :ref:`nrf_security` library, relocated from the sdk-nrfxlib repository.
+    It is located in the :file:`ext/oberon` directory and the :file:`subsys/nrf_security` directory.
```

Comment:
I fined it a bit confusing saying that the nrf_security is placed in the ext/oberon. I find it more clear to say that it is located in subsys/nrf_security. nrf_security is just using ext/oberon, right?

Reply:
I agree.

nrf_security is just using ext/oberon, yes.